pages/configuration/web_interface.rst

```diff
@@ -142,14 +142,12 @@ NGINX
           proxy_set_header X-Forwarded-Host $host;
           proxy_set_header X-Forwarded-Server $host;
           proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-          proxy_set_header X-Graylog-Server-URL http://$server_addr/api;
+          proxy_set_header X-Graylog-Server-URL http://$server_name/api;
```

Comment:
If `$server_name` is being used, we also need to add back the `server_name` directive.